src/main/java/com/google/api/codegen/transformer/TestCaseTransformer.java

```diff
@@ -67,7 +67,11 @@ public TestCaseView createTestCaseView(
               methodContext.getTypeTable(),
               methodConfig.getPageStreaming().getResourcesFieldConfig());
     } else if (methodConfig.isLongRunningOperation()) {
-      clientMethodName = namer.getAsyncApiMethodName(method, methodConfig.getVisibility());
+      if (clientMethodType == ClientMethodType.OperationOptionalArrayMethod) {
+        clientMethodName = namer.getApiMethodName(method, methodConfig.getVisibility());
+      } else {
+        clientMethodName = namer.getAsyncApiMethodName(method, methodConfig.getVisibility());
```

Comment:
I agree - the else case is to continue supporting NodeJS and Java, which use getAsyncApiMethodName (as they did before this change). I had to introduce the check because PHP uses the standard getApiMethodName for LRO methods.

Perhaps I could add a new method, namer.getLroApiMethodName?